ui-tests/tests/jupyterlite_xeus.spec.ts

```diff
@@ -44,11 +46,40 @@ test.describe('General Tests', () => {
     );
   });
 
+  test('should support the same kernel from a second environment', async ({
+    page
+  }) => {
+    await page.goto('lab/index.html');
+
+    const xpython = page
+      .locator('[title="Python 3.13 (XPython) [env2]"]')
```

Comment:
```suggestion
      .locator('[title="Python 3.13 (XPython) [env1]"]')
```

Since `bqplot` is installed in `env1` and that test imports bqplot, maybe that should be the one to start

Reply:
The previous test uses `env1` and confirms that `bqplot` can be imported, so this is the same test but with `env2` to confirm that it cannot be imported. I should probably add a note explaining this.

Reply:
Ah got it! Makes sense, thanks :D